src/core/MidiInterface.hpp

```diff
@@ -13,25 +13,25 @@ using namespace rack;
  */
 
 struct MidiInWrapper : RtMidiIn {
-	std::unordered_map<uint, std::list<std::vector<unsigned char>>> idMessagesMap;
-	std::unordered_map<uint, std::list<double>> idStampsMap;
-	uint uuid_c = 0;
-	uint subscribers = 0;
```

Comment:
Note: (Regarding `uint`). It's not defined by default by all compilers, see [here](https://stackoverflow.com/questions/3552094/c-uint-unsigned-int-int)